source/SkiaSharp.Build.targets

```diff
@@ -60,7 +59,12 @@
     Generate the assembly versioning attributes from the VERSIONS.txt file.
     ===================================================================================================================
   -->
-  <Target Name="_GenerateAssemblyVersionInfo" BeforeTargets="CoreCompile" Condition=" '$(SkipGenerateAssemblyVersionInfo)' != 'true' ">
+  <PropertyGroup Condition=" '$(SkipGenerateAssemblyVersionInfo)' != 'true' ">
+    <_GenerateAssemblyVersionInfoBeforeTargets>
+      CoreCompile
+    </_GenerateAssemblyVersionInfoBeforeTargets>
+  </PropertyGroup>
+  <Target Name="_GenerateAssemblyVersionInfo" BeforeTargets="$(_GenerateAssemblyVersionInfoBeforeTargets)" Condition=" '$(SkipGenerateAssemblyVersionInfo)' != 'true' ">
```

Comment:
To avoid issues when setting up tasks that may not run, use a property group. This will allow for "skipped" targets to never reach the queue.